pages/index.html

```diff
@@ -156,6 +171,22 @@ <h2>Plugin gallery</h2>
       </header>
     </section>
     <hr>
+    <section id="media">
+      <header>
+        <h2>Media</h2>
+        <strong>POLAR</strong>Express: Watch our latest talk for a comprehensible explanation of architecture and configuration.
```

Comment:
```suggestion
        Watch our latest talk for a comprehensive explanation of architecture and configuration.
```

1. Removed joke. (I don't get it.)
2. "comprehensible" => "comprehensive" (I assume you meant to spell out the latter.)

Reply:
1. It's a short and concise presentation
2. I meant comprehensible, but I don't mind if we change it.